pandas/_libs/lib.pyx

```diff
@@ -107,7 +109,10 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 def is_scalar(val: object) -> bint:
     """
     Return True if given value is scalar.
-
+    
+    Parameters
+    ----------
+    val : input argument of any type
     This includes:
     - numpy array scalar (e.g. np.int64)
```

Comment:
Can you generate the html version of this docstring? `./doc/make.py html --single=path.to.this.function`

I think this won't render properly